src/components/Timeline/TimelineNote.test.tsx

```diff
@@ -0,0 +1,128 @@
+import Wrapper from "@test/wrapper";
+import { render, screen } from "@testing-library/react";
+import React from "react";
+
+import TimelineNote from "./TimelineNote";
+
+const wrapperFriendlyDate = new Date("2018-08-07T14:30:40+00:00").toISOString();
+
+describe("TimelineNote", () => {
+  it("renders user", () => {
+    render(
+      <Wrapper>
+        <TimelineNote
+          app={null}
+          user={{
+            avatar: null,
+            id: "1",
+            email: "test@test.com",
+            firstName: "Test",
+            lastName: "User",
+            __typename: "User",
+          }}
+          date={wrapperFriendlyDate}
+          message="Note"
+          hasPlainDate={false}
+        />
+      </Wrapper>,
+    );
+
+    expect(screen.getByText("Test User")).toBeInTheDocument();
+    expect(screen.getByText("Note")).toBeInTheDocument();
+    expect(screen.getByText("TU")).toBeInTheDocument();
+    expect(screen.getByText("a few seconds ago")).toBeInTheDocument();
```

Comment:
All test should follow AAA test conversion. I'll give you example please follow it in rest of tests
```suggestion
    // Arrange
    const mockedUser = { // Add type here
            avatar: null,
            id: "1",
            email: "test@test.com",
            firstName: "Test",
            lastName: "User",
            __typename: "User",
          }
          
     // Act
    render(
      <TimelineNote
        app={null}
        user={mockedUser}
        date={wrapperFriendlyDate}
        message="Note"
        hasPlainDate={false}
      />
      , { wrapper: Wrapper }
    );

    // Assert
    expect(screen.getByText("Test User")).toBeInTheDocument();
    expect(screen.getByText("Note")).toBeInTheDocument();
    expect(screen.getByText("TU")).toBeInTheDocument();
    expect(screen.getByText("a few seconds ago")).toBeInTheDocument();
```

Reply:
Thanks for the suggestion, I updated the tests